styx-api-service/src/main/java/com/spotify/styx/api/ServiceAccountUsageAuthorizer.java

```diff
@@ -253,7 +254,7 @@ private String lookupServiceAccountProjectId(String email) {
           .orElseThrow(() -> new ResponseException(Response.forStatus(
               BAD_REQUEST.withReasonPhrase("Project does not exist: " + projectId))));
 
-      final List<String> members = policy.getBindings().stream()
+      final List<String> members = Optional.ofNullable(policy.getBindings()).orElse(Collections.emptyList()).stream()
```

Comment:
This looks duplicate. Can we move it into `memberStatus`, and have `memberStatus(principalEmail, policy)` instead?

Reply:
I will add tests but would prefer doing massage before calling the method. It is the matter of contract, I think.

Reply:
Plus, it looks the same, but they are actually different policy/binding classes.

Reply:
Yeah I wanted to do that code deduplication but it's different types so doesn't get much nicer than this.